### ISSUE TITLE
Rest Api > Totals information > 500 internal error

### ISSUE THREAD

**Issue by Detzler** [magento/magento2#14541]
### Preconditions

1. M2.2.2
2. PHP7.1
3. Only Tablerate method activated

### Steps to reproduce

1. Log in
2. Add a product to cart
3. Open cart page
4. Use shipping estimation
5. Klick on the radio button of an estimated shipping method

### Expected result

1. The totals block refreshes

### Actual result

1. Totals block don't refresh
2. The Api request to _/rest/default/V1/carts/mine/totals-information_ fails with a 500 response

The response contains:

```
{"messages":{"error":[{"code":500,"message":"Server internal error. See details in report api/747721818448"}]}}
```

var/report/api/747721818448:
```
"Fatal Error: 'Uncaught TypeError: Argument 1 passed to MagentoQuoteModelCartTotals::setExtensionAttributes() must be an instance of MagentoQuoteApiDataTotalsExtensionInterface, instance of MagentoQuoteApiDataAddressExtension given, called in htdocs/vendor/magento/framework/Api/DataObjectHelper.php on line 125 and defined in htdocs/vendor/magento/module-quote/Model/Cart/Totals.php:592
Stack trace:
#0 htdocs/vendor/magento/framework/Api/DataObjectHelper.php(125): MagentoQuoteModelCartTotals->setExtensionAttributes(Object(MagentoQuoteApiDataAddressExtension))
#1 htdocs/vendor/magento/framework/Api/DataObjectHelper.php(85): MagentoFrameworkApiDataObjectHelper->_setDataValues(Object(MagentoQuoteModelCartTotals), Array, 'MagentoQuoteA...')
#2 htdocs/vendor/magento/module-quote/Model/Cart/CartTotalRepository.php(103): MagentoFramew' in 'htdocs/vendor/magento/module-quote/Model/Cart/Totals.php' on line 592"
```

**Comment by nuethicadam** [comment]
We're seeing the same thing on 2.2.3, doesn't seem to stop you from proceeding to checkout but it does make the cart totals block have a perpetual loading icon.

**Comment by ghost** [comment]
Hi @Detzler thank you for your report.
We've acknowledged the issue and added to our backlog.

**Comment by bscbodyscience** [comment]
Are there a fix for this?

**Comment by pietervanuhm** [comment]
> Hi @Detzler thank you for your report.
> We've acknowledged the issue and added to our backlog.

Here the same issue, after some composer updates i get this same thing. Is there a manual fix for 2.2.3 ?

**Comment by pietervanuhm** [comment]
@[bscbodyscience](/bscbodyscience), After some google if found this commit and you can fix this issue with this. https://github.com/ekuusela/magento2/commit/aa535ea5d4bf78915bddd4387a9d3c4b39943eea

**Comment by magento-engcom-team** [comment]
Hi @Detzler. 

Thank you for your report and collaboration!

The related internal Jira ticket `MAGETWO-94717` was closed as `non-reproducible` in `2.2.x`.
It means that Magento team either unable to reproduce this issue using provided _Steps to Reproduce_ from the **_Description section_** on clean or the issue has been already fixed in the scope of other tasks.

But if you still run into this problem please update or provide additional information/steps/preconditions in the **_Description section_** and reopen this issue.
